bigchaindb/common/transaction.py

```diff
@@ -13,6 +23,11 @@
 from bigchaindb.common.utils import serialize
 
 
+SUPPORTED_CRYPTOCONDITION_TYPES = ('threshold-sha-256', 'ed25519-sha-256')
```

Comment:
I like that we are explicit with what we support and don't support but on the other hand this is only used for testing purposes.